gnosis/safe/safe.py

```diff
@@ -822,7 +817,10 @@ def estimate_tx_operational_gas(self, data_bytes_length: int):
         threshold = self.retrieve_threshold()
         return 15000 + data_bytes_length // 32 * 100 + 5000 * threshold
 
-    @cache
+    @cached(
+        cache=LRUCache(maxsize=int(environ.get("CONTRACT_CACHE_SIZE", 1024 * 100))),
```

Comment:
Are these parenthesis ok? I mean, maybe I'm missing something, but should the `1024 * 100` part be an argument for `environ.get()`?

Reply:
It is the default value if the environment variable is not defined.
Eg:
```
# Python program to explain os.environ object


# importing os module
import os

# Method 1
# Print the value of
# 'MY_HOME' environment variable
print("MY_HOME:", os.environ.get('MY_HOME', "Environment variable does not exist"))


# Method 2
try:
	print("MY_HOME:", os.environ['MY_HOME'])
except KeyError:
	print("Environment variable does not exist")
```